common/Dockerfile.onbuild-buster

```diff
@@ -16,6 +16,7 @@
 ARG baseimage=astronomerinc/ap-airflow:1.10.7-buster
 
 FROM ${baseimage}
+SHELL ["/bin/bash", "-o", "pipefail", "-e", "-u", "-x", "-c"]
```

Comment:
I am not 100% sure on why this is needed on why I had to explicitly set it here since the base image too has this set but without it, I got the following error and the packages (like git) were not installed:

https://github.com/astronomer/issues/issues/1964

```
❯ astro dev start
Env file ".env" found. Loading...
Sending build context to Docker daemon    191kB
Step 1/4 : FROM astronomerio/ap-airflow:2.0.0-1.dev3-buster-onbuild
# Executing 7 build triggers
 ---> Running in 5d913c3de7a4
Removing intermediate container 5d913c3de7a4
 ---> Running in d65158eab62c
/bin/sh: 1: [[: not found
Removing intermediate container d65158eab62c
 ---> Running in 17c8fc558cd3
```

Reply:
cc @ashb If you have any ideas on why this would have occurred. In any case, this should be harmless

Reply:
I suspect it might not propogate from three base image as it's build time only